src/simplewallet/simplewallet.cpp

```diff
@@ -2704,6 +2704,35 @@ bool simple_wallet::set_device_name(const std::vector<std::string> &args/* = std
   return true;
 }
 
+bool simple_wallet::set_export_format(const std::vector<std::string> &args/* = std::vector<std::string()*/)
+{
+  if (args.size() < 2)
+  {
+    fail_msg_writer() << tr("Export format not specified");
+    return false;
+  }
+
+  if (args[1] == "Hex" || args[1] == "hex")
```

Comment:
`if (boost::algorithm::iequals(args[1], "hex"))`, etc., covers all the variants.

Reply:
I have forgotten that we already depend on `boost::algorithm` :) . You are right that it's a better option.